package.json

```diff
@@ -36,20 +36,20 @@
   },
   "scripts": {
     "build": "rm -rf dist && tsc -p ./tsconfig.build.json",
-    "lint": "tslint --project ./tsconfig.json --format stylish",
```

Comment:
The NPM scripts have been simplified in favor of passing in any arguments for specific task manually, since they were mostly aliases anyways. This also removes the dependencies between unrelated tasks such as linting and testing, which are now handled by their own jobs on CI.